test/integration/lib/philipsHue_server.js

```diff
@@ -160,9 +104,112 @@ var philipsHue_server = (function() {
      });
    });
   }
+   
+  // Callback methods
+
+  _server.use(function (req, res, next) { 
+    // foxbox does not have the json header, so need to 
+    // inject it here so the message body can be properly captured
+    req.headers['content-type'] = 'application/json';
+    last_received_cmd = req.method + ':' + req.originalUrl;
+    next();
+  });
+
+  _server.get('/', function (req, res) {
+    res.status(200).sendFile(
+      path.join(__dirname,'/html/philips_initial.html'));  
+  });
+
+  _server.get('/api/:foxboxId/', function (req, res) {   
+      
+     if (foxboxId === undefined) {
+      if (auth_status === false) {
+        foxboxId = req.params.foxboxId;
+        res.sendStatus(200);  
+      }
+      else if (auth_status === true) {
+        res.status(200).send([{
```

Comment:
Follow up from https://github.com/fxbox/foxbox/pull/508#discussion_r65592362

Even though the HTTP request was valid, I believe you need to show there is something unexpected in the request. In this case, the user is forbidden the access, which corresponds to [an error 403](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_Error). 

If the HTTP request failed for an unlisted reason, then it would be a generic 400, IMO.